src/main/java/com/revature/domain/QuestionEval.java

```diff
@@ -0,0 +1,138 @@
+package com.revature.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import javax.persistence.*;
+
+@Entity
+@Table(name="ie_question_eval")
+public class QuestionEval {
+
+	@Id
+	@Column(name="qe_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="questionEvalSeq")
+	@SequenceGenerator(allocationSize=1, name="questionEvalSeq", sequenceName="question_eval_seq")
+	private Integer id;
+	
+	@Column(name="qe_communication_score")
+	private Integer communicationScore;
+	
+	@Column(name="qe_knowledge_score")
+	private Integer knowledgeScore;
+	
+	@ManyToOne(fetch = FetchType.EAGER)
+	@JoinColumn(name="qe_eid")
+	private Eval eval;
+	
+	@ManyToOne(fetch = FetchType.EAGER)
+	@JoinColumn(name="qe_qpid")
+	private QuestionPool questionPool;
+	
+	public QuestionEval(){}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)